frontend/src/components/login-form/index.tsx

```diff
@@ -54,43 +55,81 @@ export const LoginForm: React.SFC<RouteComponentProps<{ lang: string }>> = ({
 
   return (
     <Box
-      as="form"
       sx={{
-        maxWidth: "container",
-        mx: "auto",
-      }}
-      method="post"
-      onSubmit={(e: React.FormEvent<HTMLFormElement>) => {
-        e.preventDefault();
-
-        login({ variables: formState.values });
+        px: 2,
       }}
     >
-      {errorMessage && <div>{errorMessage}</div>}
-
-      <Label {...label("email")}>
-        <FormattedMessage id="login.email" />
-      </Label>
-      <Input
-        {...email("email")}
-        placeholder="guido@python.org"
-        required={true}
-        type="email"
-      />
-
-      <Label htmlFor="login-password">
-        <FormattedMessage id="login.password" />
-      </Label>
-      <Input
-        id="login-password"
-        {...password("password")}
-        required={true}
-        type="password"
-      />
-
-      <Button size="medium" palette="primary" isLoading={loading} type="submit">
-        <FormattedMessage id="login.loginButton" />
-      </Button>
+      {location && location.state && location.state.message && (
```

Comment:
nit: I think we can use the ? operator now: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
Changed!